run.py

```diff
@@ -19,9 +21,12 @@ def install_and_import(package):
 
 if __name__ == '__main__':
     install_and_import('rlbot')
-    from rlbot.utils import public_utils
-
-    if public_utils.is_safe_to_upgrade():
+    from rlbot.utils import public_utils, logging_utils
+    import logging
```

Comment:
this is not needed if you are using the rlbot logging utils

Reply:
Removed import for logging and used the RLBot log level instead